app/controllers/user/applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+class User::ApplicationsController < ApplicationController
+	before_action :ensure_either_broker_or_ff
+
+	def index
+		if current_user.bids.exists?
+			@applied_bids = current_user.bids.all
+		end
```

Comment:
I'm wondering if we need to check if bids exists here - does removing this cause an error? :)